nix/overlay.nix

```
@@ -7,7 +7,11 @@ self: super: {
     ];
   }));
 
-  pythonForAnsible = (self.python3.withPackages (_: self.ansible.requiredPythonModules));
+  pythonForAnsible = (self.python3.withPackages (_: self.ansible.requiredPythonModules ++ [
+                                                                                             super.python3Packages.boto
+                                                                                             super.python3Packages.boto3
+                                                                                             super.python3Packages.six
+                                                                                           ] ));
```

Or maybe it has to be changed this way
```suggestion
  pythonForAnsible = (self.python3.withPackages (_: self.ansible_with_libs.requiredPythonModules));
```